docs/MessageList.md

```diff
@@ -233,12 +233,23 @@ You must use the following properties in your XML to change your MessageListView
 | `app:streamAttachmentTitleTextColor`      | color                | #026DFE |
 | `app:streamAttachmentTitleTextStyle`      | normal, bold, italic | bold    |
 | `app:streamAttachmentDescriptionTextSize` | dimension            | 11sp    |
-| `app:streamAttachmentDescriptionTextColor`| color                | #989898 |
+| `app:streamAttachmentDescriptionTextColor`| color                | DKGRAY  |
 | `app:streamAttachmentDescriptionTextStyle`| normal, bold, italic | normal  |
 | `app:streamAttachmentFileSizeTextSize`    | dimension            | 12sp    |
-| `app:streamAttachmentFileSizeTextColor`   | color                | #989898 |
+| `app:streamAttachmentFileSizeTextColor`   | color                | DKGRAY  |
```

Comment:
`DKGRAY` is not mentioned anywhere else, so I would replace `DKGRAY` with `stream_gray_dark`.